documentation/book/proc-downgrading-brokers-older-kafka.adoc

```diff
@@ -8,9 +8,9 @@
 
 This procedure describes how you can downgrade a {ProductName} Kafka cluster to a lower (previous) version of Kafka, such as downgrading from {KafkaVersionHigher} to {KafkaVersionLower}.
 
-NOTE: Downgrading is *not possible* if the new version has ever used a `log.message.format.version` that is not supported by the previous version, including when the default value for `log.message.format.version` is used.
-
-For example, this resource can be downgraded to Kafka version {KafkaVersionLower} because the `log.message.format.version` has not been changed:
+====
```

Comment:
Important blocks should be formatted like this:

IMPORTANT: my text goes here

https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/

Reply:
I was thinking how to avoid it being a note type of thing that might be skipped. Here's how I have it formatted now. 

[IMPORTANT]
====
Downgrading is *not possible* if the new version has ever used a `log.message.format.version` that is not supported by the previous version, including when the default value for `log.message.format.version` is used. For example, this resource can be downgraded to Kafka version {KafkaVersionLower} because the `log.message.format.version` has not been changed:

[source,yaml,subs=attributes+]
----
apiVersion: v1alpha1
kind: Kafka
spec:
  # ...
  kafka:
    version: {KafkaVersionHigher}
    config:
      log.message.format.version: "{LogMsgVersLower}"
      # ...
----

The downgrade would not be possible if the `log.message.format.version` was set at `"{LogMsgVersHigher}"` or a value was absent (so that the parameter took the default value for a {KafkaVersionHigher} broker of {LogMsgVersHigher}).
====